pynsot/commands/cmd_protocols.py

```diff
@@ -147,7 +147,8 @@ def add(ctx, auth_string, attributes, circuit, device, description, interface, s
     data = ctx.params
 
     if interface is None and circuit is None:
-        raise click.UsageError('Must have interface "-i" / "--interface" or circuit "-c" / "--circuit" populated')
+        raise click.UsageError('''Must have interface "-i" / "--interface" or
```

Comment:
That looks a little funky. Perhaps putting the string on its own line might be more readable?

e.g.
```
raise click.UsageError(
    'blah blah blah
)
```